soap_test.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+
+# Try to dump SOAP vectors (features) for each atom of each molecule
+
+import sys
+import numpy as np
+from ase.io import read as ase_read
+from rascal.representations import SphericalInvariants as SOAP
+
+# What is this integer? Unit? Reasonable range to explore?
+max_radial = 6
+# What is this integer? Unit? Reasonable range to explore?
+max_angular = 6
+# what is a power spectrum?
```

Comment:
The power spectrum is the standard SOAP descriptor, but there are many other that can be constructed. Power spectrum is a 3-body descriptor, while the radial spectrum is a 2-body representation. 

Bi-Spectrum is the name of the 4-body representation, and then we just refer to them with the body order.